mage_ai/data_cleaner/statistics/calculator.py

```diff
@@ -209,6 +213,40 @@ def __statistics_overview(self, series, col):
                     dates.sort_values().iloc[math.floor(len(dates) / 2)].isoformat()
                 )
                 data[f'{col}/min'] = dates.min().isoformat()
+            elif column_type == ColumnType.TEXT:
+                text_series = series_non_null
+                num_chars = text_series.apply(lambda string: len(string))
```

Comment:
nit: you can just do `text_series.str.len()`

Reply:
Fixed